src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -441,47 +572,115 @@ public int getID()
     }
 
     /**
-     * Increment the mobs killed by this colony.
-     * <p>
-     * Will award achievements for mobs killed.
+     * Get the amount of statistic.
+     *
+     * @param statistic the statistic.
+     * @return amount of statistic.
+     */
+    private int getStatisticAmount(@NotNull String statistic)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 11, down to the maximum allowed 6. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)